src/plugins/integrator.rs

```diff
@@ -295,7 +295,10 @@ type ForceComponentsChanged = Or<(
     Changed<ExternalAngularImpulse>,
 )>;
 
-fn clear_forces_and_impulses(mut forces: Query<ForceComponents, ForceComponentsChanged>) {
+/// Responsible for clearing forces and impulses on bodies.
+/// 
```

Comment:
Looks like there's some whitespace here
```suggestion
///
```

Reply:
Damn my bad